src/cache_invalidate.c

```diff
@@ -53,6 +53,7 @@ static void
 cache_invalidate_all(void)
 {
 	ts_hypertable_cache_invalidate_callback();
+	ts_bgw_job_cache_invalidate_callback();
```

Comment:
This isn't really related to this PR, and is more for @erimatnor than anyone else, but should we invalidate our function cache here?